core/src/main/java/tc/oc/pgm/action/ActionParser.java

```diff
@@ -239,6 +243,23 @@ public MessageAction parseChatMessage(Element el, Class<?> scope) throws Invalid
     return new MessageAction(text, actionbar, title);
   }
 
+  @MethodParser("play-sound")
```

Comment:
```suggestion
  @MethodParser("sound")
```
i'd avoid two-words whenever possible, it's not "place-blocks" it's just "fill", it's not "send-message", it's just "message", i'd follow the trend and call this one simply a "sound" action